### DISCUSSION TITLE
Is there a way to specify models to be used for each label in Multi-Label Prediction?

### DISCUSSION THREAD

**Discussion by mreismendes** [autogluon/autogluon#1824] [Q&A] [answered]
Hello,

I'd like to train a Multi-Label model with 3 target columns, at the same time I'd like to specify which models to use in each target.

Is there a way to do that?

I would appreciate any kind of help.

Thank you

**Comment by Innixma** [comment, marked as answer]
I don't think this is currently possible without code edits. An alternative is you train 3 separate predictors, one for each label, and specify whatever hyperparameters you want in each.

**Reply by mreismendes** [reply]
Thank you!
